tests/models/graph_tests.py

```diff
@@ -156,10 +156,6 @@ def setUpClass(cls):
         }
         models.Edge.objects.create(**edges_dict).save()
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.deleteGraph("2f7f8e40-adbc-11e6-ac7f-14109fd34195")
```

Comment:
This is interesting.  This is why the tests were blowing up for me originally, because teardown was failing right here.  This makes sense though; seems like you've chosen to just let the test suite tear down the DB than fiddling with deleting this graph.